run_all_tools.sh

```diff
@@ -30,24 +42,22 @@ function run_tfsec {
 
 # KICS
 function run_kics {
-  echo Now running KICS on all cases
-  docker pull checkmarx/kics:latest
-  docker run -t -v $PWD:/tf checkmarx/kics version | awk '{print $NF}' >version_kics.txt
-  find . -name "main.tf" -exec dirname {} \; | grep -v "\.terraform" | while read -r test_case; do
+  docker pull checkmarx/kics:nightly
```

Comment:
Would a user really use "nightly", or "latest"?

Reply:
`nightly` is one of the images available if a user wants to be up-to-date with the new queries.